src/Component.js

```diff
@@ -1,27 +1,52 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
 import * as props from "./Component.props";
+import DropDownList from "./components/DropDownList";
+import RadioButtonList from "./components/RadioButtonList"
+import { filterOptions } from "./utils/option-params";
+import styled from "styled-components";
 
-// Build a Response Component as a React Functional Component
-const Component = ({ label, text, logResults, setNextEnabled }) => {
-  const [value, setValue] = useState(text);
-  useEffect(() => {
-    setValue(text);
-    setNextEnabled(true);
-  }, [text, setNextEnabled]);
+// Container to sort out component alignment and size
+const Alignment = styled.div`
+  width: ${x => x.dropDown ? (x.width || "100%") : "fit-content"};
+  float: ${x => (x.alignment == "center") ? "" : x.alignment};
```

Comment:
any chance we can use a `display: flex` rather than floats / margins? `float` is basically awful and rarely needed since flex/grid.